docs/sources/flow/getting-started/collect-opentelemetry-data.md

```diff
@@ -144,3 +150,181 @@ otelcol.receiver.otlp "example" {
+  }
+}
+
+otelcol.processor.batch "default" {
+  output {
+    metrics = [otelcol.exporter.otlp.default.input]
+    logs    = [otelcol.exporter.otlp.default.input]
+    traces  = [otelcol.exporter.otlp.default.input]
+  }
+}
+
+otelcol.exporter.otlp "default" {
+  client {
+    endpoint = "my-otlp-grpc-server:4317"
+  }
+}
+```
+
+For more information on configuring OpenTelemetry data batching, refer to
+[otelcol.processor.batch][].
+
+## Configure an OpenTelemetry receiver
+
+Grafana Agent Flow can be configured to receive OpenTelemetry metrics, logs,
+and traces. An OpenTelemetry _receiver_ component is responsible for receiving
+OpenTelemetry data from an external system.
+
+In this task, we will use the [otelcol.receiver.otlp][] component to receive
+OpenTelemetry data over the network using the OpenTelemetry Protocol (OTLP). A
+receiver component can be configured to forward received data to other Grafana
+Agent Flow components.
+
+> Refer to the list of available [Components][] for the full list of
+> `otelcol.receiver` components that can be used to receive OpenTelemetry data.
+>
+> [Components]: {{< relref "../reference/components/" >}}
+
+To configure an `otelcol.receiver.otlp` component for receiving OpenTelemetry
+data, complete the following steps:
+
+1. Follow [Configure an OpenTelemetry
+   exporter](#configure-an-opentelemetry-exporter) to ensure received data can
+   be written to an external system.
+
+2. Optional: Follow [Configure batching](#configure-batching) to improve
+   compression and reduce the total amount of network requests.
+
+3. Add the following `otelcol.receiver.otlp` component to your configuration
+   file:
+
+   ```river
+   otelcol.receiver.otlp "LABEL" {
+     output {
+       metrics = [COMPONENT_INPUT_LIST]
+       logs    = [COMPONENT_INPUT_LIST]
+       traces  = [COMPONENT_INPUT_LIST]
+     }
+   }
+   ```
+
+    1. Replace `LABEL` with a label to use for the component, such as
+       `default`. The label chosen must be unique across all
+       `otelcol.receiver.otlp` components in the same configuration file.
+
+    2. Replace `COMPONENT_INPUT_LIST` with a comma-delimited list of component
+       inputs to forward received data to. For example, to send data to an
+       existing batch processor component, use
+       `otelcol.processor.batch.PROCESSOR_LABEL.input`. To send data directly
+       to an existing exporter component, use
+       `otelcol.exporter.otlp.EXPORTER_LABEL.input`.
+
+    3. To enable receiving OTLP data over gRPC on port `4317`, add `grpc {}` to
+       your `otelcol.receiver.otlp` component.
+
+    4. To enable receiving OTLP data over HTTP on port `4318`, add `http {}` to
+       your `otelcol.receiver.otlp` component.
+
+    5. To disable one of the telemetry types, set the relevant type in the
+       `output` block to the empty list, such as `metrics = []`.
+
+The following example demonstrates configuring `otelcol.receiver.otlp` and
+sending it to an exporter:
```

Comment:
Do we want to mention that all the user has to do is point their application to the Agent's OTel http/grpc port for the signals to flow through?